### PR TITLE
Update MoCo Version and MDLM params

### PR DESCRIPTION
**Pull request by nvdreidenbach**
Update Version and MDLM options. Also remove duplicate files

### CODE REVIEW

File: sub-packages/bionemo-moco/documentation.md [hunk 1]
```diff
@@ -2227,6 +2119,28 @@ Perform a single step of MDLM DDPM step.
 
 - `Tensor` - The updated state.
 
+<a id="mocointerpolantscontinuous_timediscretemdlmMDLMget_num_steps_confidence"></a>
+
+#### get\_num\_steps\_confidence
+
+```python
+def get_num_steps_confidence(xt: Tensor)
```
Comment (polinabinder1) [inline sub-packages/bionemo-moco/documentation.md:2127]: Do you ever use the return value from this function?

File: sub-packages/bionemo-moco/tests/bionemo/moco/interpolants/continuous_time/discrete/test_mdlm.py [hunk 2]
```diff
@@ -112,6 +112,7 @@ def test_mdlm_step_confidence(mdlm, device):
     xt = data.clone()
     xt[:, 0] = noise[:, 0]
     time = time * 0 + 2 / 100
+    _ = mdlm.get_num_steps_confidence(xt)
```
Comment (polinabinder1) [inline sub-packages/bionemo-moco/tests/bionemo/moco/interpolants/continuous_time/discrete/test_mdlm.py:115]: Why is this called if the return value is unused?

Comment (nvdreidenbach) [reply sub-packages/bionemo-moco/tests/bionemo/moco/interpolants/continuous_time/discrete/test_mdlm.py]: I added an explicit test here to make sure the value is 1

File: sub-packages/bionemo-moco/src/bionemo/moco/interpolants/discrete_time/continuous/ddpm.py [hunk 3]
```diff
@@ -385,9 +388,10 @@ def step_noise(
         recip_sqrt_alpha_t = pad_like(recip_sqrt_alpha_t, xt)
         var = pad_like(var, xt)
 
-        x_next = recip_sqrt_alpha_t * (xt - eps_factor * eps_hat) + nonzero_mask * var.sqrt() * torch.randn_like(
-            eps_hat
-        ).to(model_out.device)
+        x_next = (
```
Comment (polinabinder1) [inline sub-packages/bionemo-moco/src/bionemo/moco/interpolants/discrete_time/continuous/ddpm.py:391]: Are there test cases for the steps?

Comment (nvdreidenbach) [reply sub-packages/bionemo-moco/src/bionemo/moco/interpolants/discrete_time/continuous/ddpm.py:391]: Thats done in the notebook but here temperature is default to one so there is no difference.

File: sub-packages/bionemo-moco/src/bionemo/moco/interpolants/continuous_time/discrete/mdlm.py [hunk 4]
```diff
@@ -202,14 +202,15 @@ def _subs_parameterization(self, logits: Tensor, xt: Tensor) -> Tensor:
         logprobs[unmasked_indices, xt[unmasked_indices]] = 0  # Unmasked token remains unchanged
         return logprobs
 
-    def step(self, logits, t, xt, dt) -> Tensor:
+    def step(self, logits: Tensor, t: Tensor, xt: Tensor, dt: Tensor, temperature: float = 1.0) -> Tensor:
```
Comment (polinabinder1) [inline sub-packages/bionemo-moco/src/bionemo/moco/interpolants/continuous_time/discrete/mdlm.py:205]: Similarly: do any tests need to be updated around this?

Comment (nvdreidenbach) [reply sub-packages/bionemo-moco/src/bionemo/moco/interpolants/continuous_time/discrete/mdlm.py:205]: This was just missing doc strings so no change other than temperature which doesn't have a deterministic outcome so all low temperature sampling methods are tested in the notebooks to verify accuracy. Automating those tests is in scope for next month

**Review by pstjohn — APPROVED** [review]
We also have a mkdocs page that builds some of this API documentation; might be worth trying to integrate that into the docs at some point?

**Review by trvachov — APPROVED** [review]

**Review by jstjohn — APPROVED** [review]
Approved but added the notebook tests flag since you have changes to notebooks.
